test/lib/www/load-images.html

```diff
@@ -0,0 +1,6 @@
+<html>
+<body>
+    <img src="logo.png"/>
+    <img src="phantomjs.png"/>
```

Comment:
Please don't use XMLisms (`<img/>`) in a file that isn't XML.  (Please *do* add `<!doctype html>` at the top of this file.)